drivers/hdc1000/hdc1000.c

```diff
@@ -72,6 +76,11 @@ int hdc1000_init(hdc1000_t *dev, const hdc1000_params_t *params)
     }
     i2c_release(dev->p.i2c);
 
+    /* set initial value for hdc device */
+    dev->last_read_time = 0xFFFFFFFF;
+    dev->temp_cached = 0;
+    dev->hum_cached  = 0;
+
```

Comment:
Are there any advantages to hold this in the device type? I personally prefer @haukepetersen's variant to keep the device type untouched and initialize:
```
#if HDC1000_RENEW_INTERVAL
static int16_t temp_chached, hum_cached;
static uint32_t last_read = 0;
#endif
```

Reply:
Actually I don't think we need to modify the device type if we don't use multiple hdc1000 sensors for a single node. I undo the change.